### PR TITLE
fix: Improve README (no-changelog)

### PR DESCRIPTION
**Pull request by OpenSourceSimon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,14 +36,18 @@ changes [here](https://github.com/n8n-io/n8n/blob/master/packages/cli/BREAKING-C
...
 
-Execute: `npx n8n`
+`npx n8n`
+
+This command will download everything that is needed to start n8n. You can then access n8n and start building workflows by opening [http://localhost:5678](http://localhost:5678).
 
 ## n8n cloud
 
-Sign-up for an [n8n cloud](https://www.n8n.io/cloud/) account.
+Sign-up for a [n8n cloud](https://www.n8n.io/cloud/) account.
```
Comment (netroy) [inline README.md:48]: since the IPA pronunciation of `n8n` is `ɛn eɪt ɛn`, `an` is the correct article here.

Comment (OpenSourceSimon) [reply README.md:48]: Okay, thanks! Didn't know that

```suggestion
Sign-up for an [n8n cloud](https://www.n8n.io/cloud/) account.
```

File: README.md [hunk 2]
```diff
@@ -70,7 +74,7 @@ quickly that all the good ones I could think of were already taken. So, in the e
 'node-' in the sense that it uses a Node-View and that it uses Node.js and '-mation' for 'automation' which is
 what the project is supposed to help with. However, I did not like how long the name was and I could not
 imagine writing something that long every time in the CLI. That is when I then ended up on 'n8n'." - **Jan
-Oberhauser, Founder and CEO, n8n.io**
+Oberhauser, Founder and CEO of n8n.io**
```
Comment (netroy) [inline README.md:77]: can you please revert this line?

**Review by netroy — CHANGES_REQUESTED** [review]
Thanks for doing this 🙏🏽. 
Can revert the last line, and then we can merge this.

**Review by netroy — APPROVED** [review]

**Comment by janober** [comment]
Got released with `n8n@0.214.1`

**Pull request merged**